README.md

```diff
@@ -1967,6 +1975,100 @@ port-channel per port hash-distribution. Valid values are 'adaptive', 'fixed' or
 port-channel per port load-defer. Valid values are true, false or 'default'. This property is not supported on (Nexus 5|6k)
 
 --
+### Type: cisco_itd_device_group
+
+Manages configuration of ITD device group
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I2(1)        | 1.3.0                  |
```

Comment:
ITD works properly only on dublin image. The camden image has a different version which is outdated and I was asked to move to dublin only. So I tested this only on dublin